app/scenes/channel/channel.js

```diff
@@ -113,17 +113,23 @@ export default class Channel extends React.PureComponent {
             return <Text>{'Waiting on channel'}</Text>;
         }
 
+        let teamId = currentChannel.team_id;
+        if (currentChannel.type === Constants.DM_CHANNEL) {
+            teamId = currentTeam.id;
+        }
+
         return (
             <KeyboardAvoidingView
                 behavior='padding'
                 style={{flex: 1, backgroundColor: theme.centerChannelBg}}
+                keyboardVerticalOffset={65}
```

Comment:
Is this needed everywhere we have a `KeyboardAvoidingView`? Perhaps we could wrap it in a custom component so we don't need to remember to add it all over